spec/acceptance/pe_status_check_spec.rb

```diff
@@ -284,6 +285,27 @@ class {'pe_status_check':
         expect(result.stdout).to match(%r{false})
         run_shell('touch -d "1 day ago"  /opt/puppetlabs/server/pe_build')
       end
+      context 'S0035' do
+        # verify pre-conditions are true
+        it 'returns true before adding an invalid module' do
```

Comment:
would it be possible to redo the testing in the format:

 ```
 it 'if S0035 conditions for false are met' do
set false
checkfalse
reset conditions
      end
```

We don't need to check true here as the main test block does that by default and it would be good to keep the test format the same for all tests